Lib/glyphsLib/interpolation.py

```diff
@@ -79,24 +84,89 @@ def build_designspace(masters, master_dir, out_dir, instance_data):
     return writer.path, instance_files
 
 
-def add_masters_to_writer(writer, ufos):
+# TODO: Use AxisDescriptor from fonttools once designSpaceDocument has been
+# made part of fonttools. https://github.com/fonttools/fonttools/issues/911
+# https://github.com/LettError/designSpaceDocument#axisdescriptor-object
+AxisDescriptor = namedtuple('AxisDescriptor', [
+    'minimum', 'maximum', 'default', 'name', 'tag', 'labelNames', 'map'])
+
+
+def get_axes(masters, instances):
+    # According to Georg Seifert, Glyphs 3 will have a better model
+    # for describing variation axes.  The plan is to store the axis
+    # information globally in the Glyphs file. In addition to actual
+    # variation axes, this new structure will probably also contain
+    # stylistic information for design axes that are not variable but
+    # should still be stored into the OpenType STAT table.
+    #
+    # We currently take the minima and maxima from the instances, and
+    # have hard-coded the default value for each axis.  We could be
```

Comment:
I don't like that we are hardcoding the default axis value here.
If the weight always has a default of 400, and my font has no master with that default value because, say, it only has two Light and Bold as masters, then `varLib`  would raise an `AssertionError`, since it's expecting to find the "base" master, i.e. one which has default locations (0 in the normalized space) along all the axes:

https://github.com/fonttools/fonttools/blob/932b938d7d80fe9182730dbc29e51fe6af3c7de5/Lib/fontTools/varLib/__init__.py#L535-L542

In theory varLib could generate such a base master given the default values in `<axes>`, but that's not the point here. We need a way to specify these defaults, and we can't wait for Glyphs 3 to be out.

Can't we define a private custom parameter (named I don't now) to store this default locations for the three weight/width and custom axes?

/cc @brawer @behdad @schriftgestalt